src/lib/components/multiselect/MultiSelect.component.tsx

```diff
@@ -105,7 +107,7 @@ function MultiSelectItem(props: ItemProps) {
         {typeof isFavorite === 'boolean' && onFavoriteClick && (
           <Button
             inverted={true}
-            icon={<i className={`${isFavorite ? 'fas' : 'far'} fa-star`} />}
```

Comment:
I don't get, it's star that is used there ?

Reply:
Yes, I forgot to change it :/

Reply:
Yup, to mark favorites in multi-select